plugins/toolkit/jetbrains-core/src/software/aws/toolkits/jetbrains/services/codemodernizer/state/CodeModernizerSessionState.kt

```diff
@@ -29,6 +25,14 @@ class CodeModernizerSessionState {
     var transformationSummary: TransformationSummary? = null
     var currentJobId: JobId? = null
 
+    companion object {
```

Comment:
as a side node, all of these moves with no changes makes this harder to review

Reply:
Refactor is inevitable. What is the best way to mitigate this @rli ? 

We are looking for feature branch PR's in the future, but the end PR result will still be a monolithic PR.

Reply:
smaller PRs into a feature branch and the merge back to main should not require much review